x-pack/plugins/fleet/common/openapi/components/schemas/fleet_settings_enrollment_response.yaml

```diff
@@ -0,0 +1,42 @@
+title: Fleet settings response
+type: object
+properties:
+  fleet_server: 
+    type: object
+    properties:
+      agent_policies: 
```

Comment:
I found that property a little confusing, I guess it's used when you want to add a Fleet server, but what do you think of having a `fleet_server_agent_policies` at the top level as I do not think there is a direct relationship between `fleet_server` and the agent policies.

Reply:
agreed, will change

Reply:
so this `agent_policies` array will only ever contain fleet server agent policies.

when there is no `?agentPolicyId` specified in the query, it will return all available fleet server policies for the user to choose from for enrolling a fleet server.

when there is a `?agentPolicyId` specified, if that ID is a fleet server policy, it will return that policy aka array length 1. if that ID is not a fleet server policy, this array will be empty.

the `fleet_server.has_active` returns a boolean based on if there is a healthy fleet server enrolled that uses any of the `agent_policies`

Reply:
What do you think of simply calling it `policies`? As Nicolas said, it's a bit confusing to read `agent_policies` in that context. In other places we use `policy_id`, as a similar approach.

Reply:
that could work 👍